gulpfile.js

```diff
@@ -806,14 +806,15 @@ function performBuild(watch) {
  */
 function checkBinarySize(compiled) {
   const file = compiled ? './dist/v0.js' : './dist/amp.js';
-  const size = compiled ? '75.1kB' : '332.3kB';
+  const size = compiled ? '76.1kB' : '332.3kB';
   const cmd = `npx bundlesize -f "${file}" -s "${size}"`;
   log(green('Running ') + cyan(cmd) + green('...\n'));
   const p = exec(cmd);
   if (p.status != 0) {
     log(red('ERROR:'), cyan('bundlesize'), 'found that amp.js/v0.js has ' +
         'exceeded its size cap. This is part of a new effort to reduce ' +
         'AMP\'s binary size (#14392). Please contact @choumx for assistance.');
+    process.exit(p.status);
```

Comment:
Do you want to do this during local dev as well? Perhaps gate this on `process.env.TRAVIS`?